include/CSVAtomicReader.hpp

```diff
@@ -0,0 +1,58 @@
+#ifndef __READERS_H__
+#define __READERS_H__
+
+#include "AtomicDataReader.hpp"
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+class CSVAtomicReader : public AtomicDataReader {
+public:
+  CSVReader(std::string filepath) : AtomicDataReader(filepath) { Read(); };
+  virtual void Read() final {
+    std::fstream file(m_filepath, std::ios::in);
+    if (file.is_open()) {
+      std::string line;
+      getline(file, line);
+      if (line == "DEFAULT") {
+        T_idx = 0;
+        rate_idx = 1;
+      }
+      std::vector<std::vector<std::string>> content;
```

Comment:
Does it build like this?
I would think `content` would need to be declared before the `if`, since it's referenced on L37